tests/virt_autotest/pvusb_run.pm

```diff
@@ -24,7 +25,8 @@ sub get_script_run {
         die "The PVUSB_DEVICE is not properly set in workers.ini.";
     }
     my $qa_repo = get_var("QA_HEAD_REPO", "http://dist.nue.suse.com/ibs/QA:/Head/SLE-12-SP3/");
-    my $guest   = get_var("GUEST",        "sles-12-sp3-64-fv-def-net");
+    handle_sp_in_settings_with_sp0("GUEST");
+    my $guest = get_var("GUEST", "sles-12-sp3-64-fv-def-net");
     $pre_test_cmd = $pre_test_cmd . " -w \"" . $which_usb . "\"";
     $pre_test_cmd = $pre_test_cmd . " -r $qa_repo -g $guest";
```

Comment:
@guoxuguang @alice-suse this 2 lines look very weird, could you use `.=` to append cmd string?

Reply:
Please recheck with the latest PR